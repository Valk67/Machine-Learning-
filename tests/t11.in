.xxxxxxxxx
.........x
x........x
x........x
x........x
x........x
..........
.xxxxxxxx.
.xxx..xxx.
..........
