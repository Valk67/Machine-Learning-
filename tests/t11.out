Sxxxxxxxxx
>v.......x
xv.......x
xv.......x
xv.......x
xv.......x
v<........
vxxxxxxxx.
vxxxG.xxx.
>>>>^.....
